Source/Urho3D/UI/UI.cpp

```diff
@@ -1797,7 +1797,7 @@ void UI::HandleMouseMove(StringHash eventType, VariantMap& eventData)
         if (!input->IsMouseVisible())
         {
             if (!input->IsMouseLocked())
-                cursor_->SetPosition(IntVector2(eventData[P_X].GetInt(), eventData[P_Y].GetInt()));
+                cursor_->SetPosition(IntVector2(eventData[P_X].GetInt(), eventData[P_Y].GetInt()) / GetScale());

```

Comment:
Huh. I suppose here is the bug, that could have been chaught by asserts in https://github.com/urho3d/Urho3D/pull/2520

Reply:
`int / float` instead of `IntVector / float` fixed it.